src/otk/document.py

```diff
@@ -19,8 +19,14 @@ class Omnifest:
     _osbuild_ctx: OSBuildContext
     _target: str
 
-    def __init__(self, path: pathlib.Path, target: str = "", *, warn_duplicated_defs: bool = False) -> None:
+    def __init__(self, paths: list[pathlib.Path], target: str = "", *, warn_duplicated_defs: bool = False) -> None:
         self._ctx = CommonContext(target_requested=target, warn_duplicated_defs=warn_duplicated_defs)
+
+        self._ctx = CommonContext(
```

Comment:
isn't self._ctx already defined in the same way two lines above? I guess this one can go (probably leftover?)

Reply:
Yea it is. Let me fix this.